hr_employee_holiday/views/hr_employee.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <!--
+      Copyright (C) 2018 Brainbean Apps (https://brainbeanapps.com)
+      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+    -->
+
+    <record id="hr_employee_view_form" model="ir.ui.view">
+        <field name="name">hr.employee.view.form.inherit.health</field>
+        <field name="model">hr.employee</field>
+        <field name="inherit_id" ref="hr.view_employee_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//notebook" position="inside">
+                <page string="Holidays">
+                    <group string="Allocations">
+                        <field name="leave_allocation_ids" nolabel="1">
+                            <tree editable="bottom">
+                                <field name="holiday_status_id"/>
+                                <field name="name"/>
+                                <field name="number_of_days" string="Allocated Days" sum="Total allocated days"/>
+                                <field name="state"/>
+                                <button string="Approve" name="action_approve" type="object"
+                                    icon="fa-check text-success"
+                                    states="confirm"
+                                    groups="hr_holidays.group_hr_holidays_user"/>
+                                <button string="Validate" name="action_validate" type="object"
+                                    icon="fa-check text-success"
+                                    states="validate1"
+                                    groups="hr_holidays.group_hr_holidays_manager"/>
+                                <button string="Refuse" name="action_refuse" type="object"
+                                    icon="fa-ban text-danger"
+                                    states="confirm,validate,validate1"
+                                    groups="hr_holidays.group_hr_holidays_manager"/>
+                            </tree>
+                        </field>
+                    </group>
+                    <group string="Requests">
+                        <field name="leave_request_ids" nolabel="1">
+                            <tree editable="bottom">
+                                <field name="holiday_status_id"/>
```

Comment:
and also I would suggest you to pass `employee_id` in  `context` here so It displays leave counts based on employee.
https://github.com/odoo/odoo/blob/12.0/addons/hr_holidays/models/hr_leave_type.py#L203

![screenshot](https://user-images.githubusercontent.com/16624719/50541262-d5826200-0bc7-11e9-9178-8c8f006f3434.png)

Reply:
done